packages/recoil/core/Recoil_RecoilEnv.js

```diff
@@ -12,14 +12,61 @@
 'use strict';
 
 const err = require('recoil-shared/util/Recoil_err');
+const gkx = require('recoil-shared/util/Recoil_gkx');
 
 export type RecoilEnv = {
   RECOIL_DUPLICATE_ATOM_KEY_CHECKING_ENABLED: boolean,
+  RECOIL_GKS_ENABLED_UNSTABLE: string[],
```

Comment:
Yeah, I would drop the `_UNSTABLE` here I guess if we are happy with the API.  Are you?  Maybe call it `RECOIL_GKS_ENABLE`?  Would it also make sense to have a `RECOIL_GKS_DISABLE` option?

Reply:
I was expecting the `RECOIL_GKS_ENABLE` to act as an override enable for a particular GK if a GK is present in that list.  That means no ability to disable a default GK.  No need to make things overly complex or add DISABLE at this time unless there's a real need, just trying to think through the ramifications of the API..

Reply:
Got it, that makes sense to me. I think in that case we just need to make the list effectively "append-only" so that if we include the default GKs in it they can't be removed. Then we could decide to allow removals in the future if we wanted (or implement a separate `...DISABLE`), which would be an API-safe change.